### DISCUSSION TITLE
couple of question

### DISCUSSION THREAD

**Discussion by JunkyardCat** [Serene-Arc/bulk-downloader-for-reddit#502] [Q&A]
1. If i archive q post again will it get overwritten? Example. I archive post 1 on 1/1/2020 then on 3/1/2020 some users have deleted thier comment or updated thier comment or the post got updated/deleted or the user gotten deleted. What would happen to the one i archived earlier?  
2. What are some pros or cons when saving in a specific file type like json xml yaml? I want to know so that i will stick to one type. I hate having lots of files with different types.  
3. Clone is the combination of archive and download right? Does the look and feel gonna be different if i use clone?  
4. Do you have plans to release like some kind of a viewer?  
5. I know theres like a limitation to 1000 post but is there a way to get the next 1000 post even if i have to like do a second run?
6. Are there any way to change the file naming?
7. I tried putting a absolute path for directory c:/user/username/programming/bfr but it resulted in an error. Not sure if youll be able to replicate it since i didnt capture the error message at the time

**Comment by Serene-Arc** [comment]
1. Yes posts will be overwritten.
2. There aren't really pros or cons, they can be converted between types easily. Depends what you want to use it for. JSON is pretty readable.
3. Yes, it is a combination, it will be the same as running both commands sequentially except saving bandwidth and time.
4. No, the BDFR is not a viewer, it is a downloader. A viewer would be a separate project.
5. No, the limit is with Reddit, not the BDFR. We can't do anything about it.
6. Yes, there is. Read the README, there's a section on that.
7. If it happens again, open a bug report.

**Reply by Binocularbath** [reply]
hello again

What is the best program to view the files with once downloaded
